tests/core/operations/test_spatialdata_operations.py

```diff
@@ -348,11 +348,18 @@ def test_no_shared_transformations() -> None:
     set_transformation(sdata.images[element_name], Identity(), to_coordinate_system=test_space)
 
     gen = sdata._gen_elements()
-    for _, name, obj in gen:
-        if name != element_name:
-            assert test_space not in get_transformation(obj, get_all=True)
-        else:
-            assert test_space in get_transformation(obj, get_all=True)
+    for element_type, name, obj in gen:
+        if element_type != "tables":
+            if name != element_name:
+                assert test_space not in get_transformation(obj, get_all=True)
+            else:
+                assert test_space in get_transformation(obj, get_all=True)
+
+
+# def test_init_from_elements(full_sdata):
+#     all_elements = {name: el for element_type, name, el in full_sdata._gen_elements() if element_type != "tables"}
+#     sdata = SpatialData.init_from_elements(all_elements, tables=full_sdata.table)
+#     for element_type in ["images", "labels", "points", "shapes", "tables"]:
 
```

Comment:
can we remove?

Reply:
Resolved with [dc4bc30](https://github.com/scverse/spatialdata/pull/410/commits/dc4bc30bf5d00a6b6d6450ceebf8561777b86944)